webclient/pages/servers.py

```diff
@@ -192,7 +197,7 @@ def servers(filter):
         }
 
     servers = _server_list_cache["servers"]
-    if filter != "all":
+    if filter is not None:
```

Comment:
Could be just "if filter:".
Unless you want to test startswith("")

Maybe convert "" to None, to make it easier in the template?